syntax = "proto3";
import "Session/SessionState.proto";

package IRacingTools.Models.RPC.Events;

message SessionEventData {
  enum Type {
    None = 0;
    Created = 1;
    Changed = 2;
    Removed = 3;
  }

  string id = 1;
  Type type = 2;

  string session_id = 10;
  Session.SessionType session_type = 11;

//  string session_name = 12;
}
